app/routes.py

```diff
@@ -1,2 +1,50 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response
 
+class Planet:
+    def __init__(self, id, name, description, color):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.color = color
+
+    def to_planet_dict(self):
+        return dict(
+            id = self.id,
+            name = self.name,
+            description = self.description,
+            color = self.color
+        )
+
+planets = [
+    Planet(1, "Pluto", "Small", "Blue"),
+    Planet(2, "Mercury", "Hot, probably", "Red"),
+    Planet(3, "Mars", "Medium", "Orange")
+]
+
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+@planets_bp.route("", methods=["GET"])
+
```

Comment:
👀 Don't leave extra space between the route decorator and the function being decorated.